### PR TITLE
BNH-165 - Boolean variables in tenant/tenant preferences

### PR DESCRIPTION
**Pull request by cec200wire**
Over35 has been changed to Under35; ETT/EET has been changed to NotInEET. Similar changes have occurred in the Filter method, and in the relevant front-end code.

### CODE REVIEW

File: web/ViewModels/TenantListModel.cs [hunk 1]
```diff
@@ -31,7 +25,15 @@ public TenantListModel(List<TenantDbModel> tenantList, HousingRequirementModel f
         PreviousPageRouteValues = GetRouteValueDictionary(-1);
         NextPageRouteValues = GetRouteValueDictionary(1);
     }
-    
+    public List<TenantDbModel> TenantList { get; set; }
```
[the remark this answers is not in the document]
Comment (cec200wire) [reply web/ViewModels/TenantListModel.cs:28]: They generally seem to be above, so I've switched things around in this file.

File: web/Views/Home/Privacy.cshtml [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿@{
+﻿@model dynamic
```
[the remark this answers is not in the document]
Comment (cec200wire) [reply web/Views/Home/Privacy.cshtml:1]: It seems not, but this ViewModel was showing up as broken and I wondered if that line - which I remembered existing at one stage - had been wrongly obliterated in a messy rebase.

File: web/Views/Property/PropertyInputForm/TenantPreferences.cshtml [hunk 3]
```diff
@@ -1,69 +1,69 @@
...
+    </div>
+    <div class="form-check">
+        @Html.CheckBox("AcceptsNotEET", Model.AcceptsNotInEET ?? false, new { @class = "form-check-input" })
+        <label asp-for="AcceptsNotInEET" class="form-check-label">
+            not in EET (Employment, Education, or Training)?
+        </label>
+    </div>
+    <div class="form-check">
+        @Html.CheckBox("AcceptsWithoutGuarantor", Model.AcceptsWithoutGuarantor ?? false, new { @class = "form-check-input" })
+        <label asp-for="AcceptsWithoutGuarantor" class="form-check-label">
```
[the remark this answers is not in the document]
Comment (cec200wire) [reply web/Views/Property/PropertyInputForm/TenantPreferences.cshtml:59]: I'm not certain, potentially the auto-formatting is introducing more line breaks to split long lines. The only things intentionally changed relate to either Over/Under35 or EET/ETT/NotInEET

File: web/Views/Home/Privacy.cshtml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-﻿@{
+﻿{
```
Comment (oskarwilliams378) [inline web/Views/Home/Privacy.cshtml:1]: I think this won't workk, you'll get this printed

**Review by oskarwilliams378 — CHANGES_REQUESTED** [review]

File: web/Views/Home/Privacy.cshtml (hunk 4 above)
Comment (cec200wire) [reply web/Views/Home/Privacy.cshtml:1]: My mistake, I forgot to re-add the @

**Review by oskarwilliams378 — APPROVED** [review]

**Pull request merged**